assets/base.css

```diff
@@ -1210,28 +1226,16 @@ deferred-media {
 
 .button.loading {
   color: transparent;
+  position: relative;
 }
 
-.button.loading:after {
-  animation: loading var(--duration-long) infinite linear;
-  border: 0.5rem solid rgba(var(--color-button-text), 0.4);
-  border-left: 0.5rem solid rgb(var(--color-button-text));
-  border-radius: 100%;
-  box-sizing: content-box;
-  content: '';
-  display: block;
-  height: 2rem;
+.button.loading > .loading-overlay__spinner {
```

Comment:
The spinner no longer exists on the featured product add to cart button. When I click add to cart on the featured product section, the button goes blank and I get a console error. 
The `loading-overlay__spinner` class does not exist in the featured product file. 

> Uncaught TypeError: Cannot read properties of null (reading 'classList')
    at HTMLElement.onSubmitHandler 

<img width="840" alt="Screen Shot 2021-09-10 at 12 15 06 PM" src="https://user-images.githubusercontent.com/26778637/132884926-fe61b4bb-e972-4242-a58a-a9b616a27fc4.png">